css/index.css

```diff
@@ -143,4 +192,70 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/* 
+
+About Section CSS Only
+
+*/
+.container .about-page .about-section{
+    border-bottom:1px dashed black;
+    padding-bottom:2em;
+}
+.container .about-page .card-container{
+    display:flex;
+    flex-wrap:wrap;
+    justify-content: space-between;
+}
+.container .about-page .card-container .card{
+    display:flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width:48%;
+    height:450px;
+    padding-bottom:40px;
+    padding-top:40px;
+    border-bottom:1px dashed black;
+}
+
+.container .about-page .card-container .card .learn-more-btn{
```

Comment:
this may be getting into more specificity than you really need to accomplish your goal